### ISSUE TITLE
I need to find Test Case/Suite by name/tag in ALL launches

### ISSUE THREAD

**Issue by JustasM11** [reportportal/reportportal#62]
Sorry, cannot reopen https://github.com/reportportal/reportportal/issues/45 . Submitting a new one.

**Comment by DzmitryHumianiuk** [comment]
@JustasM11 
at `All launches` you can interact only with launches.

please see this wireframe
https://github.com/reportportal/reportportal/issues/25#issuecomment-269011251

can this view cover your needs?

**Comment by JustasM11** [comment]
Yes, it can be covered by #25  if there will be possibility to filter not only by name but using tags also.
It can be covered by #53 also.

**Comment by avarabyeu** [comment]
@JustasM11 Just to make sure. Do you need it as API method or to be placed on UI?

**Comment by JustasM11** [comment]
I meant UI

**Comment by DzmitryHumianiuk** [comment]
https://github.com/reportportal/reportportal/issues/25
https://github.com/reportportal/reportportal/issues/53
https://github.com/reportportal/reportportal/issues/62

to be unified as one feature

**Comment by evjlobanova** [comment]
In version 5.0 we have added a clickable area for widgets which is based on filter.
For example Overall Statistics widget.
![image](https://user-images.githubusercontent.com/30413511/76962989-4d55bd00-6931-11ea-9f61-519b2f8179e0.png)

It shows summ information about filter.
When a user clicks on the widget.
Then system redirects user to the *list view for filter*
![image](https://user-images.githubusercontent.com/30413511/76963068-7d04c500-6931-11ea-9e21-46a0cbd9b282.png)
This view combines test cases from different launches.

So that partialy you problem can be solved, and you can find test cases between several launhces.

Such clickable area is also available for:
- Pattern widget
- Cumulative trend chart widget
- Overal statistics widget

In version 5.1 it will be available for all widgets which have statistics that based on several launhces.

**Comment by abotalov** [comment]
Our primary use case looks to be the possibility to find test items that failed for similar reasons.

Currently, Report Portal uses quite broad category names - Product Bug, Test Automation issue, etc. Even though subcategories can be created, they can't really be created for every different issue. Yet, it's interesting to see how often particular test flakiness occurs. Search by a part of the test item log across all launches would help to see all similar errors in all launches.
